main.py

```diff
@@ -117,6 +118,21 @@ def get_incidents():
     }
 
 
+@app.route("/user_reports")
+def get_user_reports():
```

Comment:
who will call this RPC? Admins or end users? For end users, we should only load approved reports.

Reply:
This problem is too complicated to solve in this PR. I will create a new issue for it.